packages/graphback/src/GraphbackGenerator.ts

```diff
@@ -36,13 +36,33 @@ export class GraphbackGenerator {
    */
   // FIXME generator options should be moved to plugin config
   public buildServer() {
-    const pluginEngine = new GraphbackPluginEngine(this.schema, this.config.global);
-    // TODO declarative plugin definition
-    const schemaConfig = this.config.plugins?.SchemaCRUD
-    const schemaCRUDPlugin = new SchemaCRUDPlugin(schemaConfig);
-    const resolverPlugin = new ResolverGeneratorPlugin(this.config.plugins?.ResolversCRUD);
-    const clientCRUDPlugin = new ClientCRUDPlugin(this.config.plugins.ClientCRUD)
-    pluginEngine.registerPlugin(schemaCRUDPlugin, resolverPlugin, clientCRUDPlugin);
+    const pluginEngine = new GraphbackPluginEngine(this.schema, { crudMethods: this.config.crud })
+
+    for (const pluginLabel of Object.keys(this.config.plugins)) {
+      let pluginName;
+      if (pluginLabel.startsWith('graphback-')) {
+        // Internal graphback plugins needs rename
+        pluginName = pluginLabel.replace('graphback-', '@graphback/codegen-')
```

Comment:
We need to document that custom plugins should not start with "graphback-"

Reply:
I will add TODO